.circleci/config.yml

```diff
@@ -160,11 +199,16 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - lint
       - test:
+          requires:
+            - lint
```

Comment:
To speed things up, I think we should run the linter check in parallel with the tests and the dep check. No reason not to tell contributors that the tests also don't pass when they have linter issues. Only the build and deploy jobs need to be guarded - they should require all lint, test, dep, etc. checks to pass.

Reply:
OK, that makes sense. I was looking at it more from the perspective of avoiding unecessary `test*` job runs since they'll have to push again anyway to fix the lint errors, at which point `test*` would be run again if we remove the requirement.

But sure, I'll remove it.

Reply:
Resolved by 1a188eff.